public/src/components/calls/calls_sidebar.jsx

```diff
@@ -6,10 +6,6 @@ import ActiveCallControl from './active_call_control';
 export default class CallsSideBar extends Component {
   constructor(props) {
     super(props);
-    const clickHandlers = ['handleStartCallClick', 'handleNextClick', 'handleStopClick'];
```

Comment:
I'm curious. Why don't we need the changes in this file anymore?

Reply:
After 'ActiveCallControl' was created all of these methods were moved from this file. 
When I was updating click handlers with the new updateAttempt method I noticed that this extra code was still here, so I removed it as an extension of the current cleanup.